webapp/packages/core-connections/src/DatabaseAuthDialog/SSHAuthForm.tsx

```diff
@@ -0,0 +1,85 @@
+/*
+ * CloudBeaver - Cloud Database Manager
+ * Copyright (C) 2020-2021 DBeaver Corp and others
+ *
+ * Licensed under the Apache License, Version 2.0.
+ * you may not use this file except in compliance with the License.
+ */
+
+import { observer } from 'mobx-react-lite';
+import React from 'react';
+import styled, { css } from 'reshadow';
+
+import { FieldCheckbox, FormGroup, InputField, InputGroup } from '@cloudbeaver/core-blocks';
+import { useTranslate } from '@cloudbeaver/core-localization';
+import type { NetworkHandlerConfig } from '@cloudbeaver/core-sdk';
+
+import type { DBAuthDialogController } from './DBAuthDialogController';
+
+interface Props {
+  controller: DBAuthDialogController;
+  SSHConfig: NetworkHandlerConfig;
+  allowSavePassword: boolean;
+}
+
+const styles = css`
+  form-container {
+    display: inline-flex;
+    flex-direction: column;
+  }
+`;
+
+export const SSHAuthForm: React.FC<Props> = observer(function SSHAuthForm({
+  controller, SSHConfig, allowSavePassword,
+}) {
+  const translate = useTranslate();
+  const disabled = controller.isAuthenticating;
+
+  if (!controller.config.networkCredentials.some(state => state.id === SSHConfig.id)) {
+    controller.config.networkCredentials.push({ ...SSHConfig });
+  }
+
+  const state = controller.config.networkCredentials.find(state => state.id === SSHConfig.id)!;
+
+  return styled(styles)(
+    <form-container as='div'>
+      <FormGroup>
+        <InputGroup>SSH</InputGroup>
```

Comment:
label should be loaded from `NetworkHandlerResource`
you can use `useMapResource` example in `SSHTab`
`const handler = useMapResource(NetworkHandlerResource, SSH_TUNNEL_ID);`